### ISSUE TITLE
Support Sass

### ISSUE THREAD

**Issue by NemoAlex** [borela/naomi#29]
Is it possible to add Sass support? I mean not just SCSS.

The [Syntax-highlighting-for-Sass](https://github.com/P233/Syntax-highlighting-for-Sass) just removed their built-in completions. So I'm finding other solution.

Thank you for such good work here.

**Comment by borela** [comment]
Definitely, I'll add it in a couple days as I am starting to use it on some of my projects.
